apps/passport-client/components/screens/FrogScreens/DexTab.tsx

```diff
@@ -2,15 +2,16 @@ import { EdDSAFrogPCD } from "@pcd/eddsa-frog-pcd";
 import _ from "lodash";
 import { useMemo } from "react";
 import styled from "styled-components";
+import { RippleLoader } from "../../core/RippleLoader";
 
 /**
  * The FrogeDex tab allows users to view their progress towards collecting all frogs.
  */
 export function DexTab({
-  possibleFrogCount,
+  possibleFrogIds,
   pcds
 }: {
-  possibleFrogCount?: number;
+  possibleFrogIds?: [number, number][];
```

Comment:
i feel like this would have been simpler to implement w/ just a list of possible frog ids rather than this range thing